packages/playground/windows/playground-win32/Playground-Win32.cpp

```diff
@@ -306,30 +317,33 @@ LRESULT CALLBACK WndProc(HWND hwnd, UINT message, WPARAM wparam, LPARAM lparam)
   return DefWindowProc(hwnd, message, wparam, lparam);
 }
 
-_Use_decl_annotations_ int CALLBACK WinMain(HINSTANCE instance, HINSTANCE, PSTR /* commandLine */, int showCmd) {
+int RunPlayground(int showCmd, bool useWebDebugger) {
   constexpr PCWSTR appName = L"React Native Playground (Win32)";
   constexpr PCWSTR windowClassName = L"MS_REACTNATIVE_PLAYGROUND_WIN32";
 
   winrt::init_apartment(winrt::apartment_type::single_threaded);
 
   WUXH::DesktopWindowXamlSource desktopXamlSource;
 
-  WNDCLASSEXW wcex = {};
-  wcex.cbSize = sizeof(WNDCLASSEX);
-  wcex.style = CS_HREDRAW | CS_VREDRAW;
-  wcex.lpfnWndProc = &WndProc;
-  wcex.cbClsExtra = DLGWINDOWEXTRA;
-  wcex.cbWndExtra = 0;
-  wcex.hInstance = instance;
-  wcex.hCursor = LoadCursor(nullptr, IDC_ARROW);
-  wcex.hbrBackground = (HBRUSH)(COLOR_WINDOW + 1);
-  wcex.lpszMenuName = MAKEINTRESOURCEW(IDC_PLAYGROUND_WIN32);
-  wcex.lpszClassName = windowClassName;
-  ATOM classId = RegisterClassEx(&wcex);
+  static ATOM classId{[windowClassName]() {
+    WNDCLASSEXW wcex = {};
+    wcex.cbSize = sizeof(WNDCLASSEX);
+    wcex.style = CS_HREDRAW | CS_VREDRAW;
+    wcex.lpfnWndProc = &WndProc;
+    wcex.cbClsExtra = DLGWINDOWEXTRA;
+    wcex.cbWndExtra = 0;
+    wcex.hInstance = WindowData::s_instance;
+    wcex.hCursor = LoadCursor(nullptr, IDC_ARROW);
+    wcex.hbrBackground = (HBRUSH)(COLOR_WINDOW + 1);
+    wcex.lpszMenuName = MAKEINTRESOURCEW(IDC_PLAYGROUND_WIN32);
+    wcex.lpszClassName = windowClassName;
+    return RegisterClassEx(&wcex);
+  }()};
   WINRT_VERIFY(classId);
```

Comment:
looks like this could happen in WinMain before calling RunPlayground and you wouldn't need to make this static if you move the class name out which should be fine as it's constexpr